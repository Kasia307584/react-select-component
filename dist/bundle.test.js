// import React from "react";
// import MySelect from "./bundle.js";
// import renderer from "react-test-renderer";

const React = require("react");
const MySelect = require("./bundle.js");
const renderer = require("react-test-renderer");

const options = [
  {
    label: "Sales",
    value: "Sales",
  },
  {
    label: "Marketing",
    value: "Marketing",
  },
  {
    label: "Engineering",
    value: "Engineering",
  },
  {
    label: "Human Resources",
    value: "Human Resources",
  },
  {
    label: "Legal",
    value: "Legal",
  },
];

const handleChange = (e) => {
  console.log(e);
};

test("First snapshot test", () => {
  const component = renderer.create(
    <MySelect options={options} onChange={handleChange} label="department" />
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});

test("Second snapshot test", () => {
  const component = renderer.create(
    <MySelect options={options} onChange={handleChange} />
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
