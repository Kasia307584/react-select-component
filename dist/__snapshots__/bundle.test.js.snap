// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot test 1`] = `
[
  <label
    htmlFor="department"
  >
    Department
  </label>,
  <select
    aria-label="Select option"
    className="form-select"
    id="department"
    onChange={[Function]}
    value=""
  >
    <option
      value="Sales"
    >
      Sales
    </option>
    <option
      value="Marketing"
    >
      Marketing
    </option>
    <option
      value="Engineering"
    >
      Engineering
    </option>
    <option
      value="Human Resources"
    >
      Human Resources
    </option>
    <option
      value="Legal"
    >
      Legal
    </option>
  </select>,
]
`;

exports[`Second snapshot test 1`] = `
<select
  aria-label="Select option"
  className="form-select"
  onChange={[Function]}
  value=""
>
  <option
    value="Sales"
  >
    Sales
  </option>
  <option
    value="Marketing"
  >
    Marketing
  </option>
  <option
    value="Engineering"
  >
    Engineering
  </option>
  <option
    value="Human Resources"
  >
    Human Resources
  </option>
  <option
    value="Legal"
  >
    Legal
  </option>
</select>
`;
